server.js

```diff
@@ -642,6 +642,7 @@ function getConfigData(req) {
   const masterUri = isOpenShift4() ? OPENSHIFT_PROXY_PATH : `https://${process.env.OPENSHIFT_HOST}`;
   const wssMasterUri = isOpenShift4() ? OPENSHIFT_PROXY_PATH : `wss://${process.env.OPENSHIFT_HOST}`;
 
+  const installedServices = process.env.INSTALLED_SERVICES || '[]';
```

Comment:
@aidenkeating yep this value is only used for OS4. Default value should be an object.

Reply:
Confirmed LGTM